### DISCUSSION TITLE
[BUG] Colors in body are lighter than the hex code.

### DISCUSSION THREAD

**Discussion by dianepettipas** [webstudio-is/webstudio-community#132] [Q&A]
You can see in the image below that the button in the header is a different color than the button in the hero section despite them both using the same hex code.

The elements are also using the same design token and have the same settings.

You can see in the second image that the color is even darker in the design panel color picker than on the button displayed on the screen in the hero.

The color in the header accurately represents the shade. The other button does not.

![image](https://github.com/webstudio-is/webstudio-community/assets/152535782/597b5f90-d750-42d9-9a9f-5dc58f7c014b)

![image](https://github.com/webstudio-is/webstudio-community/assets/152535782/0f52ea3b-2986-4e1f-9300-75b6298a3d9d)

**Comment by kof** [comment]
Please share a builder link with reproduction, without it we can't look into it.
